docs/glossary.rst

```diff
@@ -111,8 +115,11 @@ service address for the SecureDrop website.
 Read more about `onion services in Tor's glossary
 <https://support.torproject.org/onionservices/>`__.
 
-Onion Service v2
-""""""""""""""""
```

Comment:
I changed this because previously, it said "Onion Service v2" and then the first sentence started "The third generation of onion services (v3) provides stronger... " etc; I found that slightly confusing.